phys/module_microphysics_driver.F

```diff
@@ -398,7 +399,7 @@ SUBROUTINE microphysics_driver(                                          &
    INTEGER, OPTIONAL, INTENT(IN   )    :: hail, ice2 !, ccntype
 !
    INTEGER,      INTENT(IN   )    ::       ids,ide, jds,jde, kds,kde
-   INTEGER,      INTENT(IN   )    ::       ims,ime, jms,jme, kms,kme,num_scalar
+   INTEGER,      INTENT(IN   )    ::       ims,ime, jms,jme, kms,kme,num_scalar, num_sbmradar
```

Comment:
How does num_sbmradar come into this driver? This may need to be declared 'optional'.

Reply:
num_sbmradar  cannot be declared as optional because it is used as the dimension of the array sbmradar